### ISSUE TITLE
Create a reconciled dictionary as an optional output

### ISSUE THREAD

**Issue by dgarijo** [KnowledgeCaptureAndDiscovery/somef#579]
Where the output of the lists for each category contain reconciled information, reducing reduntant things.

**Comment by dgarijo** [comment]
And this is the one that should be used in the RDF - RDF-star mapping

**Comment by dgarijo** [comment]
Is this done @juanjemdIos ?

**Comment by juanjemdIos** [comment]
yes, we have been reconciling the output for some time, grouping identical values with their various sources and techniques. So, we output each value only once and list the different origins and techniques through which it was obtained.
